html/captive-portal/lib/captiveportal/PacketFence/DynamicRouting/Module/Authentication/SAML.pm

```diff
@@ -74,10 +76,8 @@ sub assertion {
 
     my ($username, $msg) = $self->source->handle_response($self->app->request->param("SAMLResponse"));
 
-    # We strip the username if the authorization source requires it.
-    if(isenabled($self->source->authorization_source->{stripped_user_name})){
-        ($username, undef) = strip_username($username);
-    }
+    # We strip the username if required
+    ($username, undef) = strip_username_if_needed($username, $pf::constants::realm::PORTAL_CONTEXT);
```

Comment:
&pf::config::util::strip_username_if_needed

Reply:
I'm not sure I really get why we should use the full name including package name

Reply:
I applied the PR on my server and got an issue with that.